pkg/plugins/backendplugin/instrumentation/instrumentation.go

```diff
@@ -61,6 +64,11 @@ func instrumentPluginRequest(ctx context.Context, cfg *config.Cfg, pluginCtx *ba
 			logParams = append(logParams, "uname", pluginCtx.User.Login)
 		}
 
+		pluginInfo, ok := getPluginInfoFromContext(ctx)
+		if ok {
+			logParams = append(logParams, "pluginVersion", pluginInfo.Version)
```

Comment:
Most if not all core plugins don't have a version set so this would be empty FYI